### PR TITLE
Adapt docs to recent config.log_level changes [ci skip]

### PR DESCRIPTION
**Pull request by robinboening**
This documentation change adapts to the code changes merged with this commit: https://github.com/rails/rails/commit/70018f87f0f382c357ba9f4203454a2059991a9a

### CODE REVIEW

**Comment by robinboening** [comment]
I am not completely certain if I understood the code change correctly. I can see `config.log_level` in the production template changed from `:debug` to `:info`. So I suppose it only changes the default production configuration in new/upgrading applications. Though, I don't know the consequence of having the instance variable `@log_level` changed in `railties/lib/rails/application/configuration.rb`. Does this change affect other environments as well?

File: guides/source/configuring.md [hunk 1]
```diff
@@ -116,8 +116,7 @@ numbers. It also filters out sensitive values of database columns when call `#in
 
 * `config.log_formatter` defines the formatter of the Rails logger. This option defaults to an instance of `ActiveSupport::Logger::SimpleFormatter` for all modes. If you are setting a value for `config.logger` you must manually pass the value of your formatter to your logger before it is wrapped in an `ActiveSupport::TaggedLogging` instance, Rails will not do it for you.
 
-* `config.log_level` defines the verbosity of the Rails logger. This option
-defaults to `:debug` for all environments. The available log levels are: `:debug`,
+* `config.log_level` defines the verbosity of the Rails logger. This option defaults to `:debug` for all modes except production, where it defaults to `:info`. The available log levels are: `:debug`,
```
Comment (eugeneius) [inline guides/source/configuring.md:119]:
```suggestion
* `config.log_level` defines the verbosity of the Rails logger. This option defaults to `:debug` for all environments except production, where it defaults to `:info`. The available log levels are: `:debug`,
```

I know we use "mode" elsewhere in this guide, but environment is the correct term.

Can you also rewrap at 80 characters?

**Comment by eugeneius** [comment]
https://github.com/rails/rails/commit/70018f87f0f382c357ba9f4203454a2059991a9a was followed up by https://github.com/rails/rails/commit/73079940111e8b85bf87953e5ef9fafeece5b5da, which flipped the internal default back to `:debug`. You're right that this is only true for new and upgrading applications, but that's what should be covered in the guide.

**Comment by robinboening** [comment]
Thanks for the link. I didn't see that later change.

File: guides/source/configuring.md (hunk 1 above)
Comment (robinboening) [reply guides/source/configuring.md]: @eugeneius I must admit I am a little bit confused about the wrapping part. Looking at the other paragraphs above and below I don't see them wrapped at any char length. Or are you asking me to pull the line up that is wrapped after
> are: `:debug`,

Comment (eugeneius) [reply guides/source/configuring.md]: Before this change the paragraph was wrapped at 80 characters, I was suggesting to apply that same formatting to the new version. You're right that it's not applied consistently throughout the file though, so pulling the next line up is also acceptable.

**Comment by eugeneius** [comment]
Thanks!

**Pull request merged**